### ISSUE TITLE
Pages from reddit.com fail to be displayed in Reader View

### ISSUE THREAD

**Issue by avaida** [mozilla/readability#124]
**Affected branches**:
Nightly 40.0a1 (2015-04-07), Aurora 39.0a2 (2015-04-07), Beta 38.0b2-build1 (20150406174117)

**Sample page showing this issue**:
http://www.reddit.com/r/AskReddit/comments/31txux/what_purchase_is_always_worth_the_money/

**Error message displayed**:
* `Failed to load article` (on 40.0a1 and 39.0a2)
* a redirection back to regular view on 38.0b2-build1

**Note**:
I think Reader View should *not* be available for reddit pages at all, since it's a comment based content system.

**Comment by leibovic** [comment]
I think we should just make sure `isProbablyReaderable` returns false here.

**Comment by n1k0** [comment]
I tend to agree, though where it gets really tricky is if we are supposed to ensure anything declared *readerable* should pass the whole actual content parsing and extraction. Egg and chicken really, as the only 100% sure way of achieving this would be to use the very same algorithm for both detection and extraction — which would defeat the whole purpose of having a lightweight detection function :(

**Comment by pdehaan** [comment]
As an aside, Reddit isn't _always_ a comment based content system.
For example, reader view works great on pages like http://www.reddit.com/r/javascript/comments/31va6b/lets_make_a_little_ai_war_game/ and http://www.reddit.com/r/tifu/comments/31vgyo/tifu_by_eating_breakfast_in_my_kitchen_without/
